hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneCluster.java

```diff
@@ -296,12 +296,12 @@ default String getBaseDir() {
   @SuppressWarnings("visibilitymodifier")
   abstract class Builder {
 
-    protected static final int DEFAULT_HB_INTERVAL_MS = 1000;
-    protected static final int DEFAULT_HB_PROCESSOR_INTERVAL_MS = 100;
-    protected static final int ACTIVE_OMS_NOT_SET = -1;
-    protected static final int ACTIVE_SCMS_NOT_SET = -1;
-    protected static final int DEFAULT_PIPELINE_LIMIT = 3;
-    protected static final int DEFAULT_RATIS_RPC_TIMEOUT_SEC = 1;
+    public static final int DEFAULT_HB_INTERVAL_MS = 1000;
+    public static final int DEFAULT_HB_PROCESSOR_INTERVAL_MS = 100;
+    public static final int ACTIVE_OMS_NOT_SET = -1;
+    public static final int ACTIVE_SCMS_NOT_SET = -1;
+    public static final int DEFAULT_PIPELINE_LIMIT = 3;
+    public static final int DEFAULT_RATIS_RPC_TIMEOUT_SEC = 1;
```

Comment:
Why do we have to make them `public`?

Reply:
`DEFAULT_HB_INTERVAL_MS` is used from `TestStorageContainerManager`, which is being moved to a different package.  Strictly speaking only that one needs to be `public`, I changed the rest for consistency.

On second look, probably `TestStorageContainerManager` should use the actual config, not the default one.